includes/utils/databaseConfig.php

```diff
@@ -5,7 +5,7 @@
 $dbpassword = "joodebb";
 
 try {
-  $pdo = new PDO($dsn, $dbusername);
+  $pdo = new PDO($dsn, $dbusername, $dbpassword);
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Hard-coded database password detected.

Storing passwords in source code can lead to security vulnerabilities. Consider using environment variables or a secure secrets management tool.